### ISSUE TITLE
Links to traces should be in the bisect bot bug comment if available

### ISSUE THREAD

**Issue by anniesullie** [catapult-project/catapult#1862]
When the bisect finds a culprit, the dashboard posts an update to the bug. At this point, the dashboard knows:

   * Which metric regressed
   * Whether that metric is a page metric with a trace
   * Whether that metric is a summary metric with subpages that have traces

Also, the bisect bot should have traces linked in its chartjson, right?

So there are several options for linking the before/after traces directly in the bug comment, which would make it much easier to debug the problem.

@dave-2 @primiano @petrcermak @qyearsley WDYT?

**Comment by qyearsley** [comment]
I can't find the trace links in the chartjson for test runs during bisect ([example run](https://build.chromium.org/p/tryserver.chromium.perf/builders/mac_hdd_perf_bisect/builds/292/steps/Reading%20chartjson%20results%20%282%29/logs/json.output)) or in the runs on chromium.perf ([example run, different test & platform](https://build.chromium.org/p/chromium.perf/builders/Linux%20Perf%20%282%29/builds/11336/steps/page_cycler.intl_hi_ru.reference/logs/json.output)). Related bug: [444887: "Upload traces from bisect results"](444887).

**Comment by qyearsley** [comment]
Oh, yep, traces are being uploaded for a subset of perf tests, and are listed in chart_data/charts/trace in the chartjson. [Example test run with trace links](https://build.chromium.org/p/chromium.perf/builders/Linux%20Perf%20%282%29/builds/11336/steps/smoothness.tough_scrolling_cases/logs/json.output).

The part of bisect that parses chartjson ([bisect_tester](https://code.google.com/p/chromium/codesearch#chromium/build/scripts/slave/recipe_modules/bisect_tester/)) could grab URLs if available and then they could be stored with the [revision state](https://code.google.com/p/chromium/codesearch#chromium/build/scripts/slave/recipe_modules/auto_bisect/perf_revision_state.py) for different revisions and potentially output in the bisect results.

**Comment by anniesullie** [comment]
Yeah, only tests with TBM would have the traces, so looks like recipe bisect is working as expected.

Is the revision state code (and the bug output) one of the areas that will get unit tests soon? I would love to add this feature, but it's a little scary not knowing whether things would break.

**Comment by primiano** [comment]
Not sure what is the right place to add this in the infrastructure, but I'd be definitely super-happy to see the link to the traces being attached to the bugs.
